planemo/commands/cmd_container_register.py

```diff
@@ -159,7 +159,14 @@ def handle_pull_request(self, ctx, name, target_filename, packages_str, tools_st
 
     def write_targets(self, ctx, target_filename, mulled_targets):
         with open(target_filename, "w") as f:
-            contents = ",".join(["%s=%s" % (t.package_name, t.version) for t in mulled_targets])
+            target_strings = list()
```

Comment:
how about `contents = ",".join(["%s=%s" % (t.package_name, t.version) if t.version else "%s" % t.package_name for t in mulled_targets])` ?